src/cli_keystore/get_keystore.js

```diff
@@ -21,7 +21,7 @@ import { existsSync } from 'fs';
 import { join } from 'path';
 
 import { Logger } from '../cli_plugin/lib/logger';
-import { getConfigDirectory, getDataPath } from '../core/server/path';
+import { getConfigDirectory, getDataPath } from '@kbn/utils';
```

Comment:
Is it okay that `getDataPath` ignores a value configured in `kibana.yml`? We've even got an issue for this https://github.com/elastic/kibana/issues/43539

Reply:
The plan is to first move this over, then resolve that issue - might wait for https://github.com/elastic/kibana/pull/76874 to assist with that.